docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -115,10 +118,15 @@ Unconference
 To lead an unconference session
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- Anyone can lead an unconference session, as long as they have a discussion idea and a willingness to encourage attendees to talk about it.
 - Look for an empty slot on the `Unconference sign-up <https://docs.google.com/spreadsheets/d/1IjA2yQWi1qVhopJ8PyG2obMVa5nodIRsggFSJACaWRk/edit?usp=sharing>`__, and add your Unconference topic. There will be a time and a table number for each slot.
 - When it’s time for your Unconference session to begin, go to **Sessions** and select the session that corresponds to your table number.
 - The previous unconference session will be in the same table. Let the attendees of the previous session know that their time is up, and your session will begin soon.
-- Once a group has entered your session, start the conversation!
+- Once a group has entered your session, start the conversation! Here are some tips:
+  - Avoid doing roundtable introductions. You might run out of time to actually start the conversation!
```

Comment:
This formatting isn't right. The nested list is [not a list at all in the preview](https://writethedocs-www--1345.org.readthedocs.build/conf/prague/2020/welcome-wagon/#to-lead-an-unconference-session).

Reply:
Oh no 🙈 
I did fix it but nested lists don't look good so I went back to having a separate subsection about asking questions during the Q&A session 👍

Reply:
It looks like you haven't pushed that change yet. I'm satisfied though, as long as it's fixed 😄

Reply:
I've pushed the changes in [this commit](https://github.com/writethedocs/www/pull/1345/commits/5e23a59a164cb0e7b3cd121d66e8495507b5d282) - and [the Netlify deployment looks ok to me](https://writethedocs-www--1345.org.readthedocs.build/conf/prague/2020/welcome-wagon/#conference-talks-and-q-a):

![image](https://user-images.githubusercontent.com/17963330/94557263-9ed6f100-025e-11eb-8543-4b7358a4ad79.png)

But it's true that I also didn't see the update if opening the build in a new tab. I can only see it if I stay in the same tab or go Incognito. Not sure what's going on here 🤔